proto/v1/sql_service.proto

```diff
@@ -14,6 +16,86 @@ service SQLService {
       body: "*"
     };
   }
+  rpc Query(QueryRequest) returns (QueryResponse) {
+    option (google.api.http) = {
+      post: "/v1/{instance=instances/*}:query"
```

Comment:
should be `name=instances/*`

see https://google.aip.dev/136#resource-based-custom-methods